vuu-ui/packages/vuu-filters/src/filter-clause/ExpandoCombobox.tsx

```diff
@@ -63,6 +65,7 @@ export const ExpandoCombobox = forwardRef(function ExpandoCombobox<
   const [value, setValue] = useState(
     valueProp === undefined ? "" : valueProp.toString(),
   );
+  const [touched, setTouched] = useState<boolean>(false);
```

Comment:
nit: don't really need the explicit typing here, it will infer the correct type from the value